src/components/html-text-input.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @format
+ * @flow
+ */
+
+import React from 'react';
+import { Platform, TextInput, KeyboardAvoidingView } from 'react-native';
+import styles from './html-text-input.scss';
+
+// Gutenberg imports
+import { serialize } from '@wordpress/blocks';
+
+type PropsType = {
+	blocks: Array<Object>,
+	parseBlocksAction: string => mixed,
+};
+
+type StateType = {
+	html: string,
+};
+
+export default class HTMLInputView extends React.Component<PropsType, StateType> {
+	state = {
```

Comment:
Should we initialize the state using a constructor? If I interpret the doc properly, that's the recommended way as per the [React doc](https://reactjs.org/docs/react-component.html#constructor)?

> Constructor is the only place where you should assign this.state directly. In all other methods, you need to use this.setState() instead.

Reply:
I took it from [this example](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html#initializing-state).
I believe I had read it too, that if the constructor is only initializing the state, it is valid to remove the constructor and init the state it in this way. Not sure if it's just a convenience.

Reply:
Perfect, let's keep it as it currently is in the PR! 👍